tests/containers/helm.pm

```diff
@@ -83,6 +84,17 @@ sub run {
 
     install_helm();
 
+    assert_script_run('curl --create-dir -vo ~/helm-test/Chart.yaml '.data_url('containers/helm-test/').'Chart.yaml');
+    assert_script_run('curl --create-dir -vo ~/helm-test/values.yaml '.data_url('containers/helm-test/').'values.yaml');
+    assert_script_run('curl --create-dir -vo ~/helm-test/templates/job.yaml '.data_url('containers/helm-test/templates/').'job.yaml');
+    assert_script_run('curl --create-dir -vo ~/helm-test/templates/NOTES.txt '.data_url('containers/helm-test/templates/').'NOTES.txt');
+
+    assert_script_run("helm install helm-test-$job_id ~/helm-test/ --values ~/helm-test/values.yaml --set job_id=$job_id");
+    assert_script_run("helm list");
+    my $pod = script_output('kubectl get pods -o name --no-headers=true | grep helm-test-$job_id');
+    script_retry("kubectl logs $pod | grep 'SUSE'", retry => 5, delay => 10);
+    assert_script_run("helm uninstall helm-test-$job_id");
```

Comment:
I think the uninstall should be added in case the test fails

Reply:
Done.